FileMetadataReading.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+import os
+from FileDiscovery import FileDiscovery
+
+
+class FileMetadateRading:
```

Comment:
Typo FileMetadate ->   FileMetadata

Reply:
Name of the class suggest it will provide data of a file while in reality it provides data of list of files